admin/faq.md

```diff
@@ -1,76 +1,58 @@
 # Frequently Asked Questions
-
-## What is Reaction Commerce?
+## General
+### What is Reaction Commerce?
 
 Reaction Commerce is an open source commerce platform that allows anyone to customize, extend, and deploy an online store or marketplace. Our codebase is available for free via [GitHub](https://github.com/reactioncommerce/reaction).
 
 We also offer [Reaction Platform](https://reactioncommerce.com/hosting), our paid platform-as-a-service. Beyond just hosting, our end-to-end system provides deployment, scaling, and support for retailers and brands.
 
-## How do I sign up for an account, login to my dashboard, and set up my domain?
+### Is there a live demo shop?
 
-You'll need to install and run Reaction locally on your computer. For more info, check out our [Installation docs](https://docs.reactioncommerce.com/reaction-docs/master/installation).    
+You can check out a live shop and use the admin tool in our [Demo Swag](https://swag.getreaction.io/) shop.
```

Comment:
You could also refer to the repos README in case somebody wants to dig into.
https://github.com/reactioncommerce/reaction-swag-shop